src/lib/cert.ts

```diff
@@ -23,20 +23,36 @@ const log = new Logger('lib/certs.ts')
 
 export let key: string;
 export let cert: string;
+export let certBundle: string;
 export let ca: string[] = [];
 export let peerID: string;
 
 export const init = async () => {
   log.debug("Reading key file");
   key = (await fs.readFile(path.join(utils.constants.DATA_DIRECTORY, utils.constants.KEY_FILE))).toString();
+  log.debug("Loaded key");
   log.debug("Reading cert file");
   cert = (await fs.readFile(path.join(utils.constants.DATA_DIRECTORY, utils.constants.CERT_FILE))).toString();
+
+  let caCertPath = path.join(utils.constants.DATA_DIRECTORY, utils.constants.CA_FILE);
+  if (await utils.fileExists(caCertPath)) {
+    log.debug("Reading CA file");
+    certBundle = (await fs.readFile(caCertPath)).toString() + cert;
+    log.debug("Loaded CA + cert");
+  } else {
+    certBundle = cert;
+    log.debug("Loaded cert");
+  }
+  log.debug("\n" + certBundle);
```

Comment:
In an early prototype of DX the CA was explicitly loaded. However, later the code was simplified as the file can just be picked up when reading the cert files. The constant was left in the utils file. The key question would be, is there a reason why it is explicitly loaded here? I was under the impression that as long as the file is placed in the correct directory, it will be picked up.

Reply:
Yeah so, if a `ca.pem` (or really so long as it is a `.pem`) is in the `peer-certs` that will definitely work like you said.

The biggest advantage of this is if a new org and DX comes along with a different  / renewed CA cert.  With this approach, the DX can advertise this CA as part of its cert so that the other existing DX's do not need to be redeployed in order to trust the new DX - essentially making CA-signed certs be treated / trusted more like self-signed certs (which is debatably not the best approach from a security perspective).

Reply:
I'm alright with removing this for now to keep the change small and give us time to think this through more, but we will need DX to support org's having separate CAs for our FireFly deployments outside Kaleido.